optimum/exporters/openvino/convert.py

```diff
@@ -32,10 +32,13 @@
 from optimum.exporters.onnx.convert import check_dummy_inputs_are_allowed
 from optimum.exporters.onnx.convert import export_pytorch as export_pytorch_to_onnx
 from optimum.exporters.onnx.convert import export_tensorflow as export_tensorflow_onnx
+from optimum.exporters.utils import _get_submodels_and_export_configs
 from optimum.utils import DEFAULT_DUMMY_SHAPES, is_diffusers_available
 from optimum.utils.save_utils import maybe_save_preprocessors
 
-from ...intel.utils.import_utils import is_nncf_available, is_optimum_version
+from ...intel.utils.import_utils import is_nncf_available
+from .base import init_model_configs
+from .model_configs import *  # noqa: F403
```

Comment:
```suggestion

```

Reply:
we need this for registering configs, the same made in neuron 
https://github.com/huggingface/optimum-neuron/blob/881d399372f750652fd53887b763106c1561b176/optimum/neuron/modeling_base.py#L31

Reply:
@echarlaix without import model_configs module decorators can not to be executed. I reworked do not use asterix for import, but is it fine if i stay 

`import optimu.exportes.openvino.model_confis` inside `optimum/exportes/openvino/__init__.py`, that should be enough for trigger decorator